pkg/controllers/job/plugins/svc/svc.go

```diff
@@ -97,6 +98,11 @@ func (sp *servicePlugin) OnJobAdd(job *batch.Job) error {
 		return err
 	}
 
+	// TODO: maybe add a flag
```

Comment:
@hzxuzhonghu can you pls add flag for this?

Reply:
@dalfos Yeah, anyway i want to know your use case.

Reply:
@hzxuzhonghu I already implemented it through calicoctl I want to avoid conflicts

Reply:
hmm... let's add a flag to this plugin :)